### PR TITLE
melonDS 0.9.1 (new cask)

### PR DESCRIPTION
**Pull request by RayyanAnsari**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [ ] `brew audit --cask {{cask_file}}` is error-free.
- [ ] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew install --cask {{cask_file}}` worked successfully.
- [ ] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/melonds.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask "melonds" do
+  version "0.9.1"
+  sha256 "5928ad133661b6ea428264894160b87cc48cdfc79461fb220c72d984613d6a62"
+
+  url "http://melonds.kuribo64.net/downloads/melonDS_0.9.1_mac64.tar.gz"
+  name "melonDS"
+  desc "Nintendo DS and DSi emulator"
+  homepage "http://melonds.kuribo64.net/"
+
+  depends_on formula: "qt"
```
Comment (vitorgalvao) [inline Casks/melonds.rb:10]: Are you sure all of these need to be installed to use the `app`? That sounds like a crummy experience, and the download page doesn’t mention it.

Comment (WaluigiWare64) [reply Casks/melonds.rb:10]: Yes, these need to be installed.

**Comment by miccal** [comment]
I have seen this error a few times lately with Casks that depend on Python:
```
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink bin/2to3
Target /usr/local/bin/2to3
already exists. You may want to remove it:
rm '/usr/local/bin/2to3'

To force the link and overwrite all conflicting files:
brew link --overwrite python@3.9

To list all files that would be deleted:
brew link --overwrite --dry-run python@3.9

Possible conflicting files are:
/usr/local/bin/2to3 -> /Library/Frameworks/Python.framework/Versions/2.7/bin/2to3
```

**Comment by WaluigiWare64** [comment]
@miccal so how would I fix it?
Is there anything I can do on my side?

**Comment by miccal** [comment]
> @miccal so how would I fix it?
> Is there anything I can do on my side?

No, it is a problem on our end I think.

**Comment by WaluigiWare64** [comment]
Thanks @miccal!
Is there anything else I need to change in the Cask file, or are we only waiting for the Python dependency fix?

**Comment by WaluigiWare64** [comment]
@miccal @vitorgalvao What should I do now?
The checks failed, but when you look at them, they didn't even run.

**Comment by WaluigiWare64** [comment]
@miccal @vitorgalvao The checks now finally have passed.
This is ready to merge!

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**